### ISSUE TITLE
OmKafka - Kafka version compatibility

### ISSUE THREAD

**Issue by AjithKumar-Raj** [rsyslog/rsyslog-doc#885]
Need kafka version compatibility metrics for rsyslog - omkafka

**Comment by AjithKumar-Raj** [comment]
Can you help me on this @rgerhards ?

**Comment by rgerhards** [comment]
There is no specific dependency as far as I know. Should work with everything. If you intend to use very old versions of kafka, you might want to check with librdkafka project, which is the connectivity layer we use.

**Comment by AjithKumar-Raj** [comment]
Thanks @rgerhards . We are currently using 0.10.1.1. We are planning to use latest stable kafka version only. Librdkakfa also supporting newer version [librdkafka-kafka compatibility](https://github.com/edenhill/librdkafka/blob/master/INTRODUCTION.md#compatibility).

If possible can we add this compatibility metrics in the [omkafka plugin document](https://www.rsyslog.com/doc/master/configuration/modules/omkafka.html)?

**Comment by rgerhards** [comment]
I don't think we should duplicate a dependency matrix. I don't like the idea to maintain that as it is completely out of our domain. Maybe just add the fact that we rely on librdkafka? Would you be up for a PR?
